dino.icu.yaml

```diff
@@ -172,6 +172,12 @@ erdtree: # erdtree.dino.icu - contact @justhypex on discord for issues/enquiries
   - exchange: mail.vortex.skyfall.dev.
     preference: 500
 
+evan:
+  ttl: 600
+  type: A
+  values:
+    - "129.213.147.149:25565"
```

Comment:
```suggestion
  value:
    - 129.213.147.149
```

Reply:
Hi alex! It's actually not for a minecraft server, that just happens to be the only port open on the server.

Reply:
Ok!